docker-for-mac/release-notes.md

```diff
@@ -16,11 +16,44 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop Community 3.2.0
+2021-03-NN
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### New
+
+- The dashboard shows a tip of the week.
```

Comment:
How about "The Docker Dashboard displays a tip of the week when you start Docker Desktop."?

Not sure if we should add more info here. @stephen-turner  @dieucao?

Reply:
Or, may be we can combine the two items?

"The Docker Dashboard opens automatically and displays a tip of the week when you start Docker Desktop."

Reply:
To me that makes it sound like it shows this week's tip every time you open it, rather than that it displays a tip once a week. So I would go with two bullets:

* The Docker Dashboard opens automatically when you start Docker Desktop.
* The Docker Dashboard displays a tip once a week.

Reply:
Alright, I'll update it